### PR TITLE
Release 0.19.0

### PR DESCRIPTION
**Pull request by zimlu02**

### CODE REVIEW

**Comment by temanbrcom** [comment]
Please, squash the commits on merge. It should also solve the ECA problem.

**Review by temanbrcom — APPROVED** [review]

File: clients/cobol-lsp-vscode-extension/README.md [hunk 1]
```diff
@@ -160,8 +160,6 @@ Syntax and semantic analysis are enabled for keywords, variables, and paragraphs
 
 The semantic analysis feature takes into account `COPY REPLACING` statements which alter the content of copybooks when checking for errors.
 
-![Semantic check before and after a REPLACING statement](/docs/images/CPYReplacing.gif)
```
Comment (grianbrcom) [inline clients/cobol-lsp-vscode-extension/README.md:163]: I think that it makes sense to delete this git file too.

**Review by grianbrcom — APPROVED** [review]
:chess_pawn:

**Review by asatklichov — APPROVED** [review]
🍰 Seems Release is a gift to me

**Pull request merged**